packages/desktop-client/src/components/reports/reports/renderCashFlowCardChartDetailed.tsx

```diff
@@ -0,0 +1,22 @@
+// @ts-strict-ignore
+import React from 'react';
+
+import { CashFlowGraph } from '../graphs/CashFlowGraph';
+
+export const renderCashFlowCardChartDetailed = (
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this utility is unnecessary. You can use `CashFlowGraph` directly instead.